source/api/commands/then.md

```diff
@@ -70,6 +72,17 @@ cy.get('button').then(($btn) => {
 
 ## Change subject
 
+### The subject is changed with another command
+
+```javascript
+cy.get('button').then(($btn) => {
+  const cls = $btn.class()
+
+  cy.wrap($btn).click().should('not.have.class', cls)
+    .find('i')
+}).should('have.class', 'spin') // assert on i element
```

Comment:
I feel this needs an explicit comment after `.find(i)` explaining that this is the last Cypress command, there is no `return` so the found `<i>` element will be yielded from `.then` callback